webpack/assets/javascripts/react_app/components/AuditsList/index.js

```diff
@@ -107,6 +120,21 @@ const AuditsList = ({ data: { audits } }) => (
             <ActionLinks allowedActions={allowedActions} />
           </Row>
 
+          <Row>
```

Comment:
I think this file is getting big.
Would you mind to separate it to a new component `AuditList/ItemRequestUuidRow`?
In this case, there is no reason to use `renderRequestUuidLink` and you can just embed it inside your new component.